### PR TITLE
Disable mPulse beacon on local

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #NUMBER

**Overall change:**

I am seeing a number of AppErrors on the mPulse dashboard. The vast majority of these ar coming from localhost. 

And as we do not yet know how to filter by environment on mPulse we should disable the beacon when on localhost.

This will also help reduce the number of beacons we are using.

**Code changes:**

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/ChartbeatAnalytics/index.test.jsx [hunk 1]
```diff
@@ -9,10 +9,14 @@ import * as testUtils from '../../lib/analyticsUtils/chartbeat';
 import * as utils from '../../lib/analyticsUtils';
 import * as amp from './amp';
 import * as canonical from './canonical';
-import { localBaseUrl } from '../../../testHelpers/config';
 import frontPageData from '../../../../data/news/frontpage';
 
 const defaultToggleState = {
+  local: {
+    chartbeatAnalytics: {
+      enabled: false,
```
Comment (dr3) [inline src/app/containers/ChartbeatAnalytics/index.test.jsx:18]: Your adding this as false, but then theres no `local` test? Can you duplicate the test one, for local please

Comment (chris-hinds) [reply src/app/containers/ChartbeatAnalytics/index.test.jsx:18]: test case added now.

**Review by dr3 — APPROVED** [review]
LGTM, needs testing

**Review by FK78 — APPROVED** [review]

**Comment by chris-hinds** [comment]
CC Issues are for tests so can be ignored.

This does not require manual testing.
